[fizz][folly] Can't build "fizz" when the feature "liburing" is enabled in "folly" (x64-linux)

Package: fizz:x64-linux@2024.01.01.00

**Host Environment**

- Host: x64-linux
- Compiler: GNU 13.2.1
-    vcpkg-tool version: 2024-03-14-7d353e869753e5609a1f1a057df3db8fd356e49d
    vcpkg-scripts version: 1921ac852 2024-03-28 (12 hours ago)

**To Reproduce**

- Generate standard vcpkg files with `vcpkg new --application`
- Edit `vcpkg.json` to add:
  - A dependency on `folly`, including the feature `liburing`
  - A dependency on `fizz`
- Install dependencies
- It is observed that `folly` builds successfully, but `fizz` (which depends on folly) does not.

Note that `fizz` builds successfuly as long as the feature `liburing` is turned off for folly.

**Failure logs**

```
-- Using cached facebookincubator-fizz-v2024.01.01.00.tar.gz.
-- Cleaning sources at /opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /opt/vcpkg/downloads/facebookincubator-fizz-v2024.01.01.00.tar.gz
-- Applying patch 0001-fix-libsodium.patch
-- Applying patch 0002-fix-libevent.patch
-- Applying patch 0003-fix-deps.patch
-- Using source at /opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean
-- Found external ninja('1.10.2').
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /usr/bin/ninja -v
    Working Directory: /opt/vcpkg/buildtrees/fizz/x64-linux-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /opt/vcpkg/buildtrees/fizz/config-x64-linux-dbg-CMakeCache.txt.log
      /opt/vcpkg/buildtrees/fizz/config-x64-linux-rel-CMakeCache.txt.log
      /opt/vcpkg/buildtrees/fizz/config-x64-linux-out.log

Call Stack (most recent call first):
  /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  /root/.cache/vcpkg/registries/git-trees/b3a03c5a987e674153efbc6e8a9ab1ed9655b0b0/portfile.cmake:23 (vcpkg_cmake_configure)
  scripts/ports.cmake:175 (include)



```
<details><summary>/opt/vcpkg/buildtrees/fizz/config-x64-linux-out.log</summary>

```
[1/2] "/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" -E chdir "../../x64-linux-dbg" "/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" "/opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean/fizz" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/opt/vcpkg/packages/fizz_x64-linux/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DBUILD_TESTS=OFF" "-DBUILD_EXAMPLES=OFF" "-DINCLUDE_INSTALL_DIR:STRING=include" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/opt/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/opt/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/opt/vcpkg" "-D_VCPKG_INSTALLED_DIR=/root/wd/scratch/fizz-fail/build/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../../x64-linux-dbg/CMakeCache.txt 
"/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" -E chdir "../../x64-linux-dbg" "/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" "/opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean/fizz" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/opt/vcpkg/packages/fizz_x64-linux/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DBUILD_TESTS=OFF" "-DBUILD_EXAMPLES=OFF" "-DINCLUDE_INSTALL_DIR:STRING=include" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/opt/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/opt/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/opt/vcpkg" "-D_VCPKG_INSTALLED_DIR=/root/wd/scratch/fizz-fail/build/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Deprecation Warning at CMakeLists.txt:4 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The CXX compiler identification is GNU 13.2.1
-- The C compiler identification is GNU 13.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/rh/gcc-toolset-13/root/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /opt/rh/gcc-toolset-13/root/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found libevent include directory: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/include
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_cored.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_core.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_extrad.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_extra.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_pthreadsd.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_pthreads.a
-- Found libevent 2.2.0 in /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux
-- Found ZLIB: optimized;/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libz.a;debug;/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libz.a (found version "1.3.1")
-- Found OpenSSL: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libcrypto.a (found version "3.2.1")
CMake Error at /opt/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package):
  By not providing "FindLibUring.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "LibUring",
  but CMake did not find one.

  Could not find a package configuration file provided by "LibUring" with any
  of the following names:

    LibUringConfig.cmake
    liburing-config.cmake

  Add the installation prefix of "LibUring" to CMAKE_PREFIX_PATH or set
  "LibUring_DIR" to a directory containing one of the above files.  If
  "LibUring" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  /opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/share/cmake-3.29/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/folly/folly-config.cmake:78 (find_dependency)
  /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/folly/vcpkg-cmake-wrapper.cmake:4 (_find_package)
  /opt/vcpkg/scripts/buildsystems/vcpkg.cmake:813 (include)
  CMakeLists.txt:49 (find_package)


-- Configuring incomplete, errors occurred!
[2/2] "/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" -E chdir ".." "/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" "/opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean/fizz" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/opt/vcpkg/packages/fizz_x64-linux" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DBUILD_TESTS=OFF" "-DBUILD_EXAMPLES=OFF" "-DINCLUDE_INSTALL_DIR:STRING=include" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/opt/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/opt/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/opt/vcpkg" "-D_VCPKG_INSTALLED_DIR=/root/wd/scratch/fizz-fail/build/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" -E chdir ".." "/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake" "/opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean/fizz" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/opt/vcpkg/packages/fizz_x64-linux" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DBUILD_TESTS=OFF" "-DBUILD_EXAMPLES=OFF" "-DINCLUDE_INSTALL_DIR:STRING=include" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/opt/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/opt/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/opt/vcpkg" "-D_VCPKG_INSTALLED_DIR=/root/wd/scratch/fizz-fail/build/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Deprecation Warning at CMakeLists.txt:4 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The CXX compiler identification is GNU 13.2.1
-- The C compiler identification is GNU 13.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/rh/gcc-toolset-13/root/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /opt/rh/gcc-toolset-13/root/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found libevent include directory: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/include
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_cored.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_core.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_extrad.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_extra.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_pthreadsd.a
-- Found libevent component: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_pthreads.a
-- Found libevent 2.2.0 in /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux
-- Found ZLIB: optimized;/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libz.a;debug;/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libz.a (found version "1.3.1")
-- Found OpenSSL: /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libcrypto.a (found version "3.2.1")
CMake Error at /opt/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package):
  By not providing "FindLibUring.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "LibUring",
  but CMake did not find one.

  Could not find a package configuration file provided by "LibUring" with any
  of the following names:

    LibUringConfig.cmake
    liburing-config.cmake

  Add the installation prefix of "LibUring" to CMAKE_PREFIX_PATH or set
  "LibUring_DIR" to a directory containing one of the above files.  If
  "LibUring" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  /opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/share/cmake-3.29/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/folly/folly-config.cmake:78 (find_dependency)
  /root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/folly/vcpkg-cmake-wrapper.cmake:4 (_find_package)
  /opt/vcpkg/scripts/buildsystems/vcpkg.cmake:813 (include)
  CMakeLists.txt:49 (find_package)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.
```
</details>

<details><summary>/opt/vcpkg/buildtrees/fizz/config-x64-linux-dbg-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: /opt/vcpkg/buildtrees/fizz/x64-linux-dbg
# It was generated by CMake: /opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//The subdirectory where binaries should be installed
BIN_INSTALL_DIR:STRING=bin

//No help, variable specified on the command line.
BUILD_EXAMPLES:UNINITIALIZED=OFF

//No help, variable specified on the command line.
BUILD_SHARED_LIBS:UNINITIALIZED=OFF

//No help, variable specified on the command line.
BUILD_TESTS:UNINITIALIZED=OFF

//Path to a program.
CMAKE_ADDR2LINE:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/addr2line

//Path to a program.
CMAKE_AR:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/ar

//Choose the type of build, options are: None Debug Release RelWithDebInfo
// MinSizeRel ...
CMAKE_BUILD_TYPE:STRING=Debug

CMAKE_CROSSCOMPILING:BOOL=OFF

//CXX compiler
CMAKE_CXX_COMPILER:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/c++

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_CXX_COMPILER_AR:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/gcc-ar

//Compiler launcher for CXX.
CMAKE_CXX_COMPILER_LAUNCHER:STRING=ccache

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_CXX_COMPILER_RANLIB:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/gcc-ranlib

//Flags used by the CXX compiler during all build types.
CMAKE_CXX_FLAGS:STRING=-fPIC

//Flags used by the CXX compiler during DEBUG builds.
CMAKE_CXX_FLAGS_DEBUG:STRING=-g

//Flags used by the CXX compiler during MINSIZEREL builds.
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the CXX compiler during RELEASE builds.
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the CXX compiler during RELWITHDEBINFO builds.
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//C compiler
CMAKE_C_COMPILER:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/cc

//A wrapper around 'ar' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_C_COMPILER_AR:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/gcc-ar

//Compiler launcher for C.
CMAKE_C_COMPILER_LAUNCHER:STRING=ccache

//A wrapper around 'ranlib' adding the appropriate '--plugin' option
// for the GCC compiler
CMAKE_C_COMPILER_RANLIB:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/gcc-ranlib

//Flags used by the C compiler during all build types.
CMAKE_C_FLAGS:STRING=-fPIC

//Flags used by the C compiler during DEBUG builds.
CMAKE_C_FLAGS_DEBUG:STRING=-g

//Flags used by the C compiler during MINSIZEREL builds.
CMAKE_C_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG

//Flags used by the C compiler during RELEASE builds.
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG

//Flags used by the C compiler during RELWITHDEBINFO builds.
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG

//Path to a program.
CMAKE_DLLTOOL:FILEPATH=CMAKE_DLLTOOL-NOTFOUND

//No help, variable specified on the command line.
CMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION:UNINITIALIZED=ON

//Flags used by the linker during all build types.
CMAKE_EXE_LINKER_FLAGS:STRING=

//Flags used by the linker during DEBUG builds.
CMAKE_EXE_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during MINSIZEREL builds.
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during RELEASE builds.
CMAKE_EXE_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during RELWITHDEBINFO builds.
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Enable/Disable output of compile commands during generation.
CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=

//No help, variable specified on the command line.
CMAKE_EXPORT_NO_PACKAGE_REGISTRY:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY:UNINITIALIZED=ON

//No help, variable specified on the command line.
CMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY:UNINITIALIZED=ON

//Value Computed by CMake.
CMAKE_FIND_PACKAGE_REDIRECTS_DIR:STATIC=/opt/vcpkg/buildtrees/fizz/x64-linux-dbg/CMakeFiles/pkgRedirects

//No help, variable specified on the command line.
CMAKE_INSTALL_BINDIR:STRING=bin

//The subdirectory where CMake package config files should be installed
CMAKE_INSTALL_DIR:STRING=lib/cmake/fizz

//No help, variable specified on the command line.
CMAKE_INSTALL_LIBDIR:STRING=lib

//Install path prefix, prepended onto install directories.
CMAKE_INSTALL_PREFIX:PATH=/opt/vcpkg/packages/fizz_x64-linux/debug

//No help, variable specified on the command line.
CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP:UNINITIALIZED=TRUE

//Path to a program.
CMAKE_LINKER:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/ld

//No help, variable specified on the command line.
CMAKE_MAKE_PROGRAM:UNINITIALIZED=/usr/bin/ninja

//Flags used by the linker during the creation of modules during
// all build types.
CMAKE_MODULE_LINKER_FLAGS:STRING=

//Flags used by the linker during the creation of modules during
// DEBUG builds.
CMAKE_MODULE_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of modules during
// MINSIZEREL builds.
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL:STRING=

//Flags used by the linker during the creation of modules during
// RELEASE builds.
CMAKE_MODULE_LINKER_FLAGS_RELEASE:STRING=

//Flags used by the linker during the creation of modules during
// RELWITHDEBINFO builds.
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO:STRING=

//Path to a program.
CMAKE_NM:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/nm

//Path to a program.
CMAKE_OBJCOPY:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/objcopy

//Path to a program.
CMAKE_OBJDUMP:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/objdump

//Value Computed by CMake
CMAKE_PROJECT_DESCRIPTION:STATIC=

//Value Computed by CMake
CMAKE_PROJECT_HOMEPAGE_URL:STATIC=

//Value Computed by CMake
CMAKE_PROJECT_NAME:STATIC=fizz

//Value Computed by CMake
CMAKE_PROJECT_VERSION:STATIC=1.0.0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_MAJOR:STATIC=1

//Value Computed by CMake
CMAKE_PROJECT_VERSION_MINOR:STATIC=0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_PATCH:STATIC=0

//Value Computed by CMake
CMAKE_PROJECT_VERSION_TWEAK:STATIC=

//Path to a program.
CMAKE_RANLIB:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/ranlib

//Path to a program.
CMAKE_READELF:FILEPATH=/opt/rh/gcc-toolset-13/root/usr/bin/readelf

//Flags used by the linker during the creation of shared libraries
// during all build types.
CMAKE_SHARED_LINKER_FLAGS:STRING=

//Flags used by the linker during the creation of shared libraries
// during DEBUG builds.
CMAKE_SHARED_LINKER_FLAGS_DEBUG:STRING=

//Flags used by the linker during the creation of shared libraries
// during MINSIZEREL builds.
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL:STRING=

...
Skipped 202 lines
...

//Path to a file.
_event_h:PATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/include

//Path to a library.
_event_lib_dbg:FILEPATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libevent_pthreadsd.a

//Path to a library.
_event_lib_rel:FILEPATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libevent_pthreads.a

//The directory containing a CMake configuration file for double-conversion.
double-conversion_DIR:PATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/double-conversion

//Value Computed by CMake
fizz_BINARY_DIR:STATIC=/opt/vcpkg/buildtrees/fizz/x64-linux-dbg

//Value Computed by CMake
fizz_IS_TOP_LEVEL:STATIC=ON

//Value Computed by CMake
fizz_SOURCE_DIR:STATIC=/opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean/fizz

//The directory containing a CMake configuration file for folly.
folly_DIR:PATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/folly

//The directory containing a CMake configuration file for gflags.
gflags_DIR:PATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/gflags

//The directory containing a CMake configuration file for glog.
glog_DIR:PATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/share/glog

//Path to a library.
pkgcfg_lib__OPENSSL_crypto:FILEPATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libcrypto.a

//Path to a library.
pkgcfg_lib__OPENSSL_dl:FILEPATH=/usr/lib64/libdl.a

//Path to a library.
pkgcfg_lib__OPENSSL_ssl:FILEPATH=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libssl.a


########################
# INTERNAL cache entries
########################

//ADVANCED property for variable: CMAKE_ADDR2LINE
CMAKE_ADDR2LINE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_AR
CMAKE_AR-ADVANCED:INTERNAL=1
//This is the directory where this CMakeCache.txt was created
CMAKE_CACHEFILE_DIR:INTERNAL=/opt/vcpkg/buildtrees/fizz/x64-linux-dbg
//Major version of cmake used to create the current loaded cache
CMAKE_CACHE_MAJOR_VERSION:INTERNAL=3
//Minor version of cmake used to create the current loaded cache
CMAKE_CACHE_MINOR_VERSION:INTERNAL=29
//Patch version of cmake used to create the current loaded cache
CMAKE_CACHE_PATCH_VERSION:INTERNAL=0
//Path to CMake executable.
CMAKE_COMMAND:INTERNAL=/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cmake
//Path to cpack program executable.
CMAKE_CPACK_COMMAND:INTERNAL=/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/cpack
//Path to ctest program executable.
CMAKE_CTEST_COMMAND:INTERNAL=/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/ctest
//ADVANCED property for variable: CMAKE_CXX_COMPILER
CMAKE_CXX_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_COMPILER_AR
CMAKE_CXX_COMPILER_AR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_COMPILER_RANLIB
CMAKE_CXX_COMPILER_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS
CMAKE_CXX_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_DEBUG
CMAKE_CXX_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_MINSIZEREL
CMAKE_CXX_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_RELEASE
CMAKE_CXX_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_CXX_FLAGS_RELWITHDEBINFO
CMAKE_CXX_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_COMPILER
CMAKE_C_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_COMPILER_AR
CMAKE_C_COMPILER_AR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_COMPILER_RANLIB
CMAKE_C_COMPILER_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS
CMAKE_C_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_DEBUG
CMAKE_C_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_MINSIZEREL
CMAKE_C_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELEASE
CMAKE_C_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELWITHDEBINFO
CMAKE_C_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_DLLTOOL
CMAKE_DLLTOOL-ADVANCED:INTERNAL=1
//Path to cache edit program executable.
CMAKE_EDIT_COMMAND:INTERNAL=/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/bin/ccmake
//Executable file format
CMAKE_EXECUTABLE_FORMAT:INTERNAL=ELF
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS
CMAKE_EXE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_DEBUG
CMAKE_EXE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_MINSIZEREL
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELEASE
CMAKE_EXE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXPORT_COMPILE_COMMANDS
CMAKE_EXPORT_COMPILE_COMMANDS-ADVANCED:INTERNAL=1
//Name of external makefile project generator.
CMAKE_EXTRA_GENERATOR:INTERNAL=
//Name of generator.
CMAKE_GENERATOR:INTERNAL=Ninja
//Generator instance identifier.
CMAKE_GENERATOR_INSTANCE:INTERNAL=
//Name of generator platform.
CMAKE_GENERATOR_PLATFORM:INTERNAL=
//Name of generator toolset.
CMAKE_GENERATOR_TOOLSET:INTERNAL=
//Test CMAKE_HAVE_LIBC_PTHREAD
CMAKE_HAVE_LIBC_PTHREAD:INTERNAL=1
//Source directory with the top level CMakeLists.txt file for this
// project
CMAKE_HOME_DIRECTORY:INTERNAL=/opt/vcpkg/buildtrees/fizz/src/4.01.01.00-5c06ddb3b6.clean/fizz
//Install .so files without execute permission.
CMAKE_INSTALL_SO_NO_EXE:INTERNAL=0
//ADVANCED property for variable: CMAKE_LINKER
CMAKE_LINKER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS
CMAKE_MODULE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_DEBUG
CMAKE_MODULE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELEASE
CMAKE_MODULE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_NM
CMAKE_NM-ADVANCED:INTERNAL=1
//number of local generators
CMAKE_NUMBER_OF_MAKEFILES:INTERNAL=1
//ADVANCED property for variable: CMAKE_OBJCOPY
CMAKE_OBJCOPY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_OBJDUMP
CMAKE_OBJDUMP-ADVANCED:INTERNAL=1
//Platform information initialized
CMAKE_PLATFORM_INFO_INITIALIZED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RANLIB
CMAKE_RANLIB-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_READELF
CMAKE_READELF-ADVANCED:INTERNAL=1
//Path to CMake installation.
CMAKE_ROOT:INTERNAL=/opt/vcpkg/downloads/tools/cmake-3.29.0-linux/cmake-3.29.0-linux-x86_64/share/cmake-3.29
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS
CMAKE_SHARED_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_DEBUG
CMAKE_SHARED_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELEASE
CMAKE_SHARED_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_INSTALL_RPATH
CMAKE_SKIP_INSTALL_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_RPATH
CMAKE_SKIP_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS
CMAKE_STATIC_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_DEBUG
CMAKE_STATIC_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELEASE
CMAKE_STATIC_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STRIP
CMAKE_STRIP-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TAPI
CMAKE_TAPI-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TOOLCHAIN_FILE
CMAKE_TOOLCHAIN_FILE-ADVANCED:INTERNAL=1
//uname command
CMAKE_UNAME:INTERNAL=/usr/bin/uname
//ADVANCED property for variable: CMAKE_VERBOSE_MAKEFILE
CMAKE_VERBOSE_MAKEFILE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CPACK_SOURCE_RPM
CPACK_SOURCE_RPM-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CPACK_SOURCE_TBZ2
CPACK_SOURCE_TBZ2-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CPACK_SOURCE_TGZ
CPACK_SOURCE_TGZ-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CPACK_SOURCE_TXZ
CPACK_SOURCE_TXZ-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CPACK_SOURCE_TZ
CPACK_SOURCE_TZ-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CPACK_SOURCE_ZIP
CPACK_SOURCE_ZIP-ADVANCED:INTERNAL=1
//Details about finding OpenSSL
FIND_PACKAGE_MESSAGE_DETAILS_OpenSSL:INTERNAL=[/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libcrypto.a][/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/include][c ][v3.2.1()]
//Details about finding Threads
FIND_PACKAGE_MESSAGE_DETAILS_Threads:INTERNAL=[TRUE][v()]
//Details about finding ZLIB
FIND_PACKAGE_MESSAGE_DETAILS_ZLIB:INTERNAL=[optimized;/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/lib/libz.a;debug;/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/libz.a][/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/include][c ][v1.3.1()]
//ADVANCED property for variable: OPENSSL_CRYPTO_LIBRARY
OPENSSL_CRYPTO_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENSSL_INCLUDE_DIR
OPENSSL_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: OPENSSL_SSL_LIBRARY
OPENSSL_SSL_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PKG_CONFIG_ARGN
PKG_CONFIG_ARGN-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PKG_CONFIG_EXECUTABLE
PKG_CONFIG_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: USE_STATIC_DEPS_ON_UNIX
USE_STATIC_DEPS_ON_UNIX-ADVANCED:INTERNAL=1
//Install the dependencies listed in your manifest:
//\n    If this is off, you will have to manually install your dependencies.
//\n    See https://github.com/microsoft/vcpkg/tree/master/docs/specifications/manifests.md
// for more info.
//\n
VCPKG_MANIFEST_INSTALL:INTERNAL=OFF
//ADVANCED property for variable: VCPKG_VERBOSE
VCPKG_VERBOSE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_INCLUDE_DIR
ZLIB_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_DEBUG
ZLIB_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_RELEASE
ZLIB_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//Making sure VCPKG_MANIFEST_MODE doesn't change
Z_VCPKG_CHECK_MANIFEST_MODE:INTERNAL=OFF
//Vcpkg root directory
Z_VCPKG_ROOT_DIR:INTERNAL=/opt/vcpkg
//linker supports push/pop state
_CMAKE_LINKER_PUSHPOP_STATE_SUPPORTED:INTERNAL=TRUE
_OPENSSL_CFLAGS:INTERNAL=-I/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../../include
_OPENSSL_CFLAGS_I:INTERNAL=
_OPENSSL_CFLAGS_OTHER:INTERNAL=
_OPENSSL_FOUND:INTERNAL=1
_OPENSSL_INCLUDEDIR:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../../include
_OPENSSL_INCLUDE_DIRS:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../../include
_OPENSSL_LDFLAGS:INTERNAL=-L/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../lib;-lssl;-lcrypto;-ldl;-pthread
_OPENSSL_LDFLAGS_OTHER:INTERNAL=-pthread
_OPENSSL_LIBDIR:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../lib
_OPENSSL_LIBRARIES:INTERNAL=ssl;crypto;dl
_OPENSSL_LIBRARY_DIRS:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../lib
_OPENSSL_LIBS:INTERNAL=
_OPENSSL_LIBS_L:INTERNAL=
_OPENSSL_LIBS_OTHER:INTERNAL=
_OPENSSL_LIBS_PATHS:INTERNAL=
_OPENSSL_MODULE_NAME:INTERNAL=openssl
_OPENSSL_PREFIX:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../..
_OPENSSL_STATIC_CFLAGS:INTERNAL=-I/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../../include
_OPENSSL_STATIC_CFLAGS_I:INTERNAL=
_OPENSSL_STATIC_CFLAGS_OTHER:INTERNAL=
_OPENSSL_STATIC_INCLUDE_DIRS:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../../include
_OPENSSL_STATIC_LDFLAGS:INTERNAL=-L/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../lib;-lssl;-lcrypto;-ldl;-pthread
_OPENSSL_STATIC_LDFLAGS_OTHER:INTERNAL=-pthread
_OPENSSL_STATIC_LIBDIR:INTERNAL=
_OPENSSL_STATIC_LIBRARIES:INTERNAL=ssl;crypto;dl
_OPENSSL_STATIC_LIBRARY_DIRS:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../lib
_OPENSSL_STATIC_LIBS:INTERNAL=
_OPENSSL_STATIC_LIBS_L:INTERNAL=
_OPENSSL_STATIC_LIBS_OTHER:INTERNAL=
_OPENSSL_STATIC_LIBS_PATHS:INTERNAL=
_OPENSSL_VERSION:INTERNAL=3.2.1
_OPENSSL_openssl_INCLUDEDIR:INTERNAL=
_OPENSSL_openssl_LIBDIR:INTERNAL=
_OPENSSL_openssl_PREFIX:INTERNAL=
_OPENSSL_openssl_VERSION:INTERNAL=
__pkg_config_arguments__OPENSSL:INTERNAL=QUIET;openssl
__pkg_config_checked__OPENSSL:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__OPENSSL_crypto
pkgcfg_lib__OPENSSL_crypto-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__OPENSSL_dl
pkgcfg_lib__OPENSSL_dl-ADVANCED:INTERNAL=1
//ADVANCED property for variable: pkgcfg_lib__OPENSSL_ssl
pkgcfg_lib__OPENSSL_ssl-ADVANCED:INTERNAL=1
prefix_result:INTERNAL=/root/wd/scratch/fizz-fail/build/vcpkg_installed/x64-linux/debug/lib/pkgconfig/../../lib
```
</details>

**Additional context**

<details open><summary>vcpkg.json</summary>

```
{
  "dependencies": [
    {
      "name": "folly",
      "features": [
        "liburing"
      ]
    },
    "fizz"
  ]
}
```
</details>

--- a/ports/folly/fix-deps.patch
+++ b/ports/folly/fix-deps.patch
@@ -2,11 +2,12 @@ diff --git a/CMake/folly-config.cmake.in b/CMake/folly-config.cmake.in
 index 1689f9a..e5d3e22 100644
 --- a/CMake/folly-config.cmake.in
 +++ b/CMake/folly-config.cmake.in
-@@ -28,10 +28,35 @@ endif()
+@@ -28,10 +28,30 @@ endif()
  set(FOLLY_LIBRARIES Folly::folly)
  
  # Find folly's dependencies
 -find_dependency(fmt)
++find_dependency(fmt CONFIG)
 +find_dependency(double-conversion CONFIG)
 +find_dependency(glog CONFIG)
 +find_dependency(gflags CONFIG)
@@ -27,12 +28,6 @@ index 1689f9a..e5d3e22 100644
 +if (NOT @CMAKE_DISABLE_FIND_PACKAGE_LZ4@)
 +    find_dependency(lz4 CONFIG)
 +endif()
-+
-+if (@WITH_liburing@)
-+    find_dependency(LibUring)
-+endif()
-+
-+find_dependency(fmt CONFIG)
  
  set(Boost_USE_STATIC_LIBS "@FOLLY_BOOST_LINK_STATIC@")
 -find_dependency(Boost 1.51.0 MODULE

--- a/ports/folly/vcpkg.json
+++ b/ports/folly/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "folly",
   "version-string": "2024.01.01.00",
-  "port-version": 2,
+  "port-version": 3,
   "description": "An open-source C++ library developed and used at Facebook. The library is UNSTABLE on Windows",
   "homepage": "https://github.com/facebook/folly",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2774,7 +2774,7 @@
     },
     "folly": {
       "baseline": "2024.01.01.00",
-      "port-version": 2
+      "port-version": 3
     },
     "font-chef": {
       "baseline": "1.1.0",

--- a/versions/f-/folly.json
+++ b/versions/f-/folly.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "e99a9ae70e66a41fc8bd159f89eba510d8dc2bc8",
+      "version-string": "2024.01.01.00",
+      "port-version": 3
+    },
+    {
       "git-tree": "6498cc8e72c9193c4aacedadf34ad4477de5c657",
       "version-string": "2024.01.01.00",
       "port-version": 2

